.github/workflows/update-pot-files.yml

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
An empty source line at the start, forever wasted :grin:

Reply:
oops, sorry for that :)